lib/ts/recipe/session/constants.ts

```diff
@@ -14,4 +14,6 @@
  */
 
 export const REFRESH_API_PATH = "/session/refresh";
-export const HANDSHAKE_INFO_FILE_PATH = "/tmp/supertokens-handshakeInfo.json";
+const SERVERLESS_CACHE_BASE_FILE_PATH =
+    process.env.SERVERLESS_CACHE_BASE_FILE_PATH === undefined ? "/tmp" : process.env.SERVERLESS_CACHE_BASE_FILE_PATH;
+export const SERVERLESS_CACHE_HANDSHAKE_INFO_FILE_PATH = `${SERVERLESS_CACHE_BASE_FILE_PATH}/supertokens-handshakeInfo.json`;
```

Comment:
Remove the .json extension and also from the other file, remove the .txt extension.